cpp/src/arrow/util/bitmap.h

```diff
@@ -387,10 +384,10 @@ class ARROW_EXPORT Bitmap : public util::ToStringOstreamable<Bitmap>,
   int64_t length() const { return length_; }
 
   /// string_view of all bytes which contain any bit in this Bitmap
```

Comment:
Maybe we should alter these method comments now? e.g:
```suggestion
  /// span of all bytes which contain any bit in this Bitmap
```